api/proto/teleport/legacy/types/types.proto

```diff
@@ -7042,6 +7042,9 @@ message PluginOktaSyncSettings {
   // that has not been yet updated and during the next update the value will be set to "org" if
   // AppID is empty or "saml_app" if not.
   string user_sync_source = 11;
+
+  // Enable Okta system log export.
+  bool enable_system_log_export = 12;
```

Comment:
I know we are quite inconsistent with godoc but in this type we start it godoc style i.e. `// EnableSystemLogExport ...`. It's also very lean. I'd like able to read what is this system log export used for and why it should be enabled/disabled.

Reply:
Auto generated code doesn't follow that because sometimes we use camel case in proto docs and the compiled go code doesn't follow the same pattern.

Reply:
Yes, but why not being consistent at least within the scope of the proto message? All other fields here follow the same pattern.

Also the other part of the comment is not addressed:

> I'd like able to read what is this system log export used for and why it should be enabled/disabled.